content/get_start/UseCase_Workflows/revit.md

```diff
@@ -0,0 +1,108 @@
+Export the model from Revit (BIM) to OpenStudio (BEM) by any of the following methods. Few differences between the export methods are highlighted below. 
+- gbXML export: All gbxml exports include essential model information, such as geometry, construction, internal gain definitions and schedules. There are some differences mentioned for each below: 
+   - Revit native gbXML 
+   - Revit Systems Analysis gbXML : Additional information included related to HVAC objects. 
+   - Autodesk Insight gbXML : Additional information included related to HVAC objects.
+
+{{% notice info %}}
+Note that the basic import feature of the OpenStudio application (i.e. the user interface) only translates a limited set of gbXML contents including geometry, constructions, thermal zones, and schedules. The translation of the remaining contents can be achieved using OpenStudio measures.
+{{% /notice %}}
+
+- Revit Systems Analysis OpenStudio model 
+    - Offers a streamlined workflow 
+    - Effectively translates most of the gbXML content into BEM using OpenStudio measures. 
+    - Various export modes available like a) Use Conceptual Masses and Building Elements (can be preferred for SD models), b) Use Rooms or Spaces (can be preferred for DD and CD models)
+
+{{% notice info %}}
+"Use Rooms or Spaces" allows the BEM model to inherit all rooms and incorporate the pre-assigned thermal zoning from the BIM model. In the BIM model multiple spaces can be associated with the same thermal zone indicating that they are controlled by a single thermostat and this is reflected in the BEM model.
+{{% /notice %}}
+
+
+ ##### **4. Apply 90.1 PRM method**
+Apply the 90.1 PRM meausre in OpenStudio by following the steps provided in the initial sections of this user guide based on the method adopted. 
+
+Key information required to run the PRM measure include space types for lighting power density, building area type for window to wall ratio, HVAC system selection and water heater system, and process loads. 
+Thus, before running the measure, it is important to check the exported BEM model for any missing information and addressing them by following the steps provided for Model Preparation below to ensure a successful measure run. 
+
+ - Model Preparation 
+     - Standard Space Type Mapping 
+       
+       Mapping of the space types is a common challenge across different export routes and design phases.   
+       - Cross validate different building area types. In PRM there are three building type categories used to determine: the baseline window-to-wall-ratio (WWR), HVAC system type, and service hot water (SHW) system type that needs to have the same type assigned to them. For example, if the section of a building model is categorized as an "Office (<=5000 ft^2)", it must not be classified as a "Multifamily" building type to determine the baseline model SHW system.
+       - If required, manually map space types from Revit to those listed in PRM to align them accurately. 
```

Comment:
- I think most of the time this will be needed so that the PRM can pick the correct LPD values. 
- Maybe provide a link here to show the list of PRM space types?

Reply:
Done in the new commit!